_episodes/03-lists.md

```diff
@@ -173,7 +175,8 @@ does not.
 > Here is a visual example of how indexing a list of lists `x` works:
 >
 > <a href='https://twitter.com/hadleywickham/status/643381054758363136'>
-> ![The first element of a list. Adapted from @hadleywickham's tweet about R lists.](../fig/indexing_lists_python.png)</a>
+> ![The first element of a list. Adapted from @hadleywickham.](../fig/indexing_lists_python.png)
+> </a>
```

Comment:
Let's rewrite this link in Markdown.

```
[![The first element of a list.
   Adapted from @hadleywickham.](../fig/indexing_lists_python.png)][hadleywickham-tweet]
```

And add the following to the bottom of the document:

```
[hadleywickham-tweet]: https://twitter.com/hadleywickham/status/643381054758363136
```